core/eth/directory/contract_names.go

```diff
@@ -6,16 +6,16 @@ package directory
 // When you add to this list, make sure you keep things in alphabetical order.
 
 const (
-	BLSOperatorStateRetriever ContractName = "OPERATOR_STATE_RETRIEVER"
-	EigenDAEjectionManager    ContractName = "EIGEN_DA_EJECTION_MANAGER"
-	RegistryCoordinator       ContractName = "REGISTRY_COORDINATOR"
-	RelayRegistry             ContractName = "RELAY_REGISTRY"
-	ServiceManager            ContractName = "SERVICE_MANAGER"
+	OperatorStateRetriever ContractName = "OPERATOR_STATE_RETRIEVER"
```

Comment:
nit, now that we've dropped the prefix, could you move this entry down so this list stays in alphabetical order?

Reply:
[0d1331e](https://github.com/Layr-Labs/eigenda/pull/1905/commits/0d1331e9b9ad41faa050eb90d93e8aa2334e4848)